benchmark/test_runner/global-concurrent-tests.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../common');
+const { it } = require('node:test');
+
+const bench = common.createBenchmark(main, {
+  n: [100, 1000, 1e4],
+}, {
+  // We don't want to test the reporter here
+  flags: ['--test-reporter=./benchmark/fixtures/empty-test-reporter.js'],
+});
+
+async function run(n) {
+  const promises = new Array(n);
+  for (let i = 0; i < n; i++) {
+    promises[i] = it(`${i}`, () => {
```

Comment:
`it('test', () => {})` and `it('test', async () => {})` behave slightly differently - it might be worth to test both